descwl/analysis.py

```diff
@@ -80,7 +80,7 @@ def __init__(self,survey,table,stamps,bounds,num_slices):
         self.bounds = bounds
         self.num_slices = num_slices
         if len(self.stamps) > 0:
-             #21 allows second partials
+             #The '21' immediately below below refers to the case when second partials are required (21 is 1 + 5 + 15, where the first term refers to the galaxy image itself, the 5 is the number of first partials excluding flux and 15 is the number of second partials exluding flux and using that partial derivatives commute.)
```

Comment:
below is typed twice

Reply:
Thanks for the catch! 
```suggestion
             #The '21' immediately below refers to the case when second partials are required (21 is 1 + 5 + 15, where the first term refers to the galaxy image itself, the 5 is the number of first partials excluding flux and 15 is the number of second partials excluding flux and using that partial derivatives commute.)
```